### DISCUSSION TITLE
Can we remove case sensitive searches? (request)

### DISCUSSION THREAD

**Discussion by Nathan269** [statamic/cms#4800] [Troubleshooting]
By default, it appears that the search feature to filter through articles on the CMS is case sensitive. Would the Statamic team be able to remove case sensitivity for the next update? [
<img width="1512" alt="Screenshot 2021-11-22 at 16 05 51" src="https://user-images.githubusercontent.com/72409095/142895502-a607a195-4186-44c8-980c-7309ab8f1d4d.png">
](url)
<img width="1512" alt="Screenshot 2021-11-22 at 16 06 01" src="https://user-images.githubusercontent.com/72409095/142895534-684fe437-b4a5-4432-ac10-3a88d77e83cc.png">

**Comment by duncanmcclean** [comment]
Have you seen this issue? #3315

**Reply by Nathan269** [reply]
Nope, I'll look into that thanks

**Reply by Nathan269** [reply]
I've looked at some of the solutions and they aren't possible for me to make direct changes to the code as it is within Statamic's code. Is there no way for yous to implement this within the next update? It's for the filter search bar within the collections? <img width="1531" alt="Screenshot 2021-11-24 at 13 23 24" src="https://user-images.githubusercontent.com/72409095/143412422-5f21079e-a9b8-457a-b53f-9a1de24cbb3a.png">

**Reply by benfurfie** [reply]
So a couple of things:

1. You may be able to add your own behaviour by extending the default function through an addon or package. It's worth looking at.
2. The core team are unlikely to remove or change functionality like this. More likely, is that they would add the ability to add a flag to the config that would change the default behaviour (case sensitive) searches. There's too much of a chance of breaking other websites.
